cmd/installer/configureStorage.go

```diff
@@ -262,15 +375,36 @@ func getImage(logger log.DebugLogger) (*image.Image, *srpc.Client, error) {
 		return nil, nil, fmt.Errorf("no image found in: %s on: %s",
 			streamName, imageServerAddress)
 	}
+	startTime = time.Now()
 	if img, err := imageclient.GetImage(client, imageName); err != nil {
 		client.Close()
 		return nil, nil, err
 	} else {
-		logger.Debugf(0, "got image: %s\n", imageName)
+		logger.Debugf(0, "got image: %s in %s\n",
+			imageName, format.Duration(time.Since(startTime)))
 		return img, client, nil
 	}
 }
 
+func getRandomKey(numBytes uint, logger log.DebugLogger) ([]byte, error) {
```

Comment:
use cryto/random instead of this

Reply:
I already have a change cooking for this, but I'm testing some odd performance behaviour. I'd like to defer this for now.